license.txt

```diff
@@ -1,4 +1,4 @@
-Copyright 2019 shopware AG
```

Comment:
how about leaving out the year like in SW5? https://github.com/shopware/shopware/blob/5.6/license.txt

Reply:
didn't take this to seriously. 
i was just looking for a half valid trivial contribution to test cla-assistant.